### DISCUSSION TITLE
How can I get the rgb tuple from mesh (load from .ply)? And turn off shadow when rendering points.

### DISCUSSION THREAD

**Discussion by CCHLENG** [marcomusy/vedo#990] [Q&A] [answered]
I have a pointcloud save in *.ply file, now I can load and show it with `load()` function.
```python
    from vedo import show, Points, load
    points = load('one-row-RGB.ply')
    points.point_size(4)   # change the size of points
    show(points).close()   # looks like rendering with shadows
```
I can get the XYZ data by `points.points()` , however, when I use `points.color()` , it only return `[1.,1.,1.]`.
**So how can I get the color data(rgb tuple) from the ply file?**
Meanwhile, when I render the pointcloud, it looks like with shadows, which make it's hard to see the color of each point.
In Meshlab, I can close shadows manually:
![image](https://github.com/marcomusy/vedo/assets/25116183/f2313d47-f50b-4276-9450-a3010c583d60)
With Vedo, from top side, the color is acceptable:
![image](https://github.com/marcomusy/vedo/assets/25116183/fe6cba33-4c97-4555-89d4-617c38e156e1)
but from another side, it's dark. **How can I make the color more clearly?**
![image](https://github.com/marcomusy/vedo/assets/25116183/ccaa70a5-42ea-4802-be55-2f59b2426571)

Thanks!

**Comment by marcomusy** [comment, marked as answer]
You can switch off lighting to have a uniform color. Try

```
pip install vedo -U
```

and

```py
from vedo import *
pc = Points(dataurl+"debug.ply")
print(pc)
print(pc.pointdata["RGB"])  # get the point colors
pc.point_size(20).lighting("off")
show(pc)
```

**Comment by CCHLENG** [comment]
It works! Thank you! 
Where can i find it in the API Documentation? I have searched by keyword"Points"and"shadow", but I didn't find the answer.

**Comment by marcomusy** [comment]
Well "shadow" is not the right word for it.. "lighting" is :)

**Comment by CCHLENG** [comment]
Get it! Thank you very much!
